functional-tests/features/wait-for-elements/wait_for_form_css.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Wait for forms
+    As a kakunin user
+    I want fill and check form fields
+
+    Scenario: Fill and check form fields
+        Given I visit the "main" page
+        When I click the ".appearForm" element
+        Then the "appearSimpleForm" page is displayed
+        When I click the ".colored" element
+        And I fill the "form" form with:
```

Comment:
You forget to set `form` to css selector

Reply:
it's the name of element ;) and css selector (related to all forms)